src/components/communications/Thread/Thread.tsx

```diff
@@ -47,9 +42,16 @@ export const Thread: FunctionComponent<ThreadProps> = ({ id, children, onSend })
         to: null,
         from: null,
         direction: 'inbound',
-        status: 'sent',
+        status: 'pending',
         sent_at: new Date().toISOString(),
-        attachments: [],
+        attachments:
+          params.attachments?.map((attachment) => ({
+            type: 'file',
+            description: attachment.file.name,
+            url: attachment.file.url,
+            metadata: attachment.metadata ?? {},
+            resource: attachment.file,
```

Comment:
this should be `.id`, no?

Reply:
No - this is the temporary message we create, where we expect attachments to have an expanded attachment. The _input_ params that go to the API only require the ID.